back/src/adapters/primary/routers/security.e2e.test.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  AgencyDtoBuilder,
+  ConventionDtoBuilder,
+  UnauthenticatedConventionRoutes,
+  expectHttpResponseToEqual,
+  unauthenticatedConventionRoutes,
+} from "shared";
+import { HttpClient } from "shared-routes";
+import { createSupertestSharedClient } from "shared-routes/supertest";
+import { AppConfigBuilder } from "../../../utils/AppConfigBuilder";
+import { InMemoryGateways, buildTestApp } from "../../../utils/buildTestApp";
+
+describe("security e2e", () => {
+  const peAgency = new AgencyDtoBuilder().withKind("pole-emploi").build();
+
+  const convention = new ConventionDtoBuilder()
+    .withAgencyId(peAgency.id)
+    .withFederatedIdentity({ provider: "peConnect", token: "some-id" })
+    .build();
+
+  let unauthenticatedRequest: HttpClient<UnauthenticatedConventionRoutes>;
+  let gateways: InMemoryGateways;
+
+  beforeEach(async () => {
+    const testApp = await buildTestApp(new AppConfigBuilder().build());
+    const request = testApp.request;
+
+    ({ gateways } = testApp);
+
+    unauthenticatedRequest = createSupertestSharedClient(
+      unauthenticatedConventionRoutes,
+      request,
+    );
+
+    gateways.timeGateway.defaultDate = new Date();
+  });
+
+  describe("check request body for HTML", () => {
+    it("400 - should throw an error if a request (POST) body parameter contains HTML", async () => {
+      const response = await unauthenticatedRequest.createConvention({
+        body: {
+          convention: {
+            ...convention,
+            businessName: "<script>alert('XSS')</script>",
+          },
+        },
+      });
+
+      expectHttpResponseToEqual(response, {
+        body: {
+          status: 400,
+          message: "Invalid request body",
+        },
+        status: 400,
+      });
+    });
+    it("200 - should not throw an error if a request (POST) body parameter does not contain HTML", async () => {
```

Comment:
ajout d'un saut de ligne

```suggestion
    });
    
    it("200 - should not throw an error if a request (POST) body parameter does not contain HTML", async () => {
```